bin/node/runtime/src/lib.rs

```diff
@@ -278,7 +278,7 @@ parameter_types! {
 parameter_types! {
 	pub const LaunchPeriod: BlockNumber = 7 * DAYS;
 	pub const VotingPeriod: BlockNumber = 7 * DAYS;
-	pub const TermDuration: BlockNumber = 14 * DAYS;
+	pub const TermDuration: BlockNumber = 10 * DAYS;
```

Comment:
I'm changing TermDuration on testnet-runtime to triple-check that shortening it doesn't break anything before deploying to mainnet (though I'm 100% sure it doesn't).